plugins/destination/clickhouse/typeconv/arrow/types/field.go

```diff
@@ -0,0 +1,95 @@
+package types
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ClickHouse/clickhouse-go/v2/lib/column"
+	"github.com/apache/arrow/go/v12/arrow"
+	"github.com/cloudquery/cloudquery/plugins/destination/clickhouse/util"
+	"github.com/cloudquery/plugin-sdk/v2/types"
+)
+
+func fieldFromColumn(col column.Interface) (*arrow.Field, error) {
```

Comment:
why not just use the string representation?

Reply:
This allows us not to parse a lot ourselves, and instead we just handle the SDK types.
Some (e.g., `Tuple`, precision & time zone for `DateTime64`, etc.) parts are still required to be parsed, but mostly we just can lean on the SDK to unwrap the nested types for us.